build/acm.mk

```
@@ -0,0 +1,205 @@
+# ACM (Advanced Cluster Management) installation targets for OpenShift
+# This file is specific to downstream OpenShift/OCP features only
+
+.PHONY: acm-install acm-mce-install acm-operator-install acm-instance-install acm-status acm-import-cluster acm-uninstall
+
+##@ ACM (OpenShift only)
+
+acm-install: acm-mce-install acm-operator-install acm-instance-install ## Install MCE, ACM operator and instance
+
+# Install MultiCluster Engine (required for ACM)
+acm-mce-install:
+	@echo "Installing MultiCluster Engine (MCE)..."
+	@echo "Creating multicluster-engine namespace..."
+	@oc create namespace multicluster-engine --dry-run=client -o yaml | oc apply -f -
+	@echo "Creating MCE OperatorGroup..."
+	@printf '%s\n' \
+		'apiVersion: operators.coreos.com/v1' \
+		'kind: OperatorGroup' \
+		'metadata:' \
+		'  name: multicluster-engine' \
+		'  namespace: multicluster-engine' \
+		'spec:' \
+		'  targetNamespaces:' \
+		'  - multicluster-engine' \
+		| oc apply -f -
+	@echo "Creating MCE Subscription..."
+	@printf '%s\n' \
+		'apiVersion: operators.coreos.com/v1alpha1' \
+		'kind: Subscription' \
+		'metadata:' \
+		'  name: multicluster-engine' \
+		'  namespace: multicluster-engine' \
+		'spec:' \
+		'  channel: stable-2.9' \
+		'  name: multicluster-engine' \
+		'  source: redhat-operators' \
+		'  sourceNamespace: openshift-marketplace' \
+		| oc apply -f -
+	@echo "Waiting for MCE operator CSV to be ready..."
+	@for i in $$(seq 1 60); do \
```

(Optional) Calling `seq` isn't required here.

```suggestion
	@for i in {1..60}; do \
```